playbooks/common/openshift-cluster/config.yml

```diff
@@ -26,6 +26,21 @@
   tags:
   - always
 
+- name: Setup firewall
+  hosts: oo_all_hosts
+  tags:
+  - always
+  tasks:
+  # This should move to intialize_facts
+  - name: set os_firewall_enabled
```

Comment:
@mtnbikenc, instead of executing the code here in openshift-cluster/config.yml, is this a candidate for std_include.yml?  If not, we need a place to initialize this a single time.

Thoughts?